scene/src/components/backpack/WearableCategoryButton.tsx

```diff
@@ -67,24 +84,21 @@ export function WearableCategoryButton({
       onMouseDown={() => {
         onClick()
       }}
+      onMouseEnter={() => {
+        state.hoveredCategory = category
+      }}
+      onMouseLeave={() => {
+        if (!(state.hoveredCategory && state.hoveredCategory !== category)) {
+          state.hoveredCategory = null
+        }
+      }}
     >
       <Icon icon={categoryIcon} iconSize={iconSize} />
-      <UiEntity
-        uiTransform={{
-          width: 124 * canvasScaleRatio,
-          height: 124 * canvasScaleRatio,
-          display: selectedURN === null ? 'none' : 'flex',
-          positionType: 'absolute',
-          position: {
-            left: 54 * canvasScaleRatio * 2
-          }
-        }}
-        uiBackground={ROUNDED_TEXTURE_BACKGROUND}
-      />
+
       <UiEntity
         uiTransform={{
           width: 116 * canvasScaleRatio,
-          height: 116 * canvasScaleRatio,
+          height: 116 * canvasScaleRatio * (127 / 120),
```

Comment:
Why this math? Can the height be 122/123? (The result of `117 * (127 / 120)` is 122.76)?